molecule/default/Dockerfile.j2

```diff
@@ -8,5 +8,5 @@ FROM {{ item.image }}
 
 # install minimal packages for debian slim images
 RUN apt-get update && \
```

Comment:
Be aware that this would probably break the build in Debian Buster, as the Python package is still being called `python`. See Prometheus Node Exporter role configuration for reference: https://github.com/idealista/prometheus_node_exporter_role/blob/develop/molecule/default/Dockerfile.j2

Reply:
Roger, fixed now